App/components/DatabaseComponent.js

```diff
@@ -0,0 +1,180 @@
+import React from "react";
+import {
+  ScrollView,
+  StyleSheet,
+  Text,
+  View,
+  TouchableOpacity,
+} from "react-native";
+import Constants from "expo-constants";
+import { FlatList, ForceTouchGestureHandler, TouchableHighlight } from "react-native-gesture-handler";
+import * as SQLite from "expo-sqlite";
+import CustomButton from "../../App code/CustomButton";
+import { Button } from "react-native";
+const db = SQLite.openDatabase("grocery_Items.db");
+
+class Items extends React.Component 
+{
+  constructor() {
+  super();
+  this.state = {
+    items: null,
+    ColorHolder: "#252525",
+  }};
+
+  componentDidMount() {
+    this.update();
+  }
+
+ markItemOwned=()=>{
+    if(this.state.ColorHolder != "coral"){
+      
+      this.setState({
+        ColorHolder: "coral"
+      })
+    }
+    else{
+      this.setState({
+        ColorHolder: "#252525"
+      })
+    }
+  }
+
+  render() {
+    const { items } = this.state;

```

Comment:
This is part of the old code that Kris and merged in. All items that are stored in database will start with color #252525 then upon pressing it will change to coral.